ingestion/src/metadata/ingestion/lineage/sql_lineage.py

```diff
@@ -64,20 +65,33 @@ def search_table_entities(
     if search_tuple in search_cache:
         return search_cache.get(search_tuple)
     try:
-        table_fqns = fqn.build(
-            metadata,
+        table_entities: Optional[List[Table]] = []
+        # search on ES first
+        fqn_search_string = build_es_fqn_search_string(
```

Comment:
Should split the try block? I am wondering if `build_es_fqn_search_string` raises if we still want to run `get_by_name()`

Reply:
We are calling `build_es_fqn_search_string ` also from `fqn.build`, so we are not going to call `get_by_name()` anyway.